### PR TITLE
feat(ct): vue3 rerender complete

### PR DESCRIPTION
**Pull request by sand4rt**
partial fix for https://github.com/microsoft/playwright/issues/15919

### CODE REVIEW

File: packages/playwright-ct-vue/registerSource.mjs [hunk 1]
```diff
@@ -187,6 +206,14 @@ window.playwrightRerender = async (rootElement, options) => {
   if (!component)
     throw new Error('Component was not mounted');
 
-  for (const [key, value] of Object.entries(options.props || {}))
+  const { slots, listeners, props } = createComponent(options);
+
+  component.slots = wrapFunctions(slots);
+  allListeners[0][1] = listeners;
```
Comment (sand4rt) [inline packages/playwright-ct-vue/registerSource.mjs:212]: @pavelfeldman Not sure if this could hurt?

Comment (pavelfeldman) [reply packages/playwright-ct-vue/registerSource.mjs:212]: I'd rather move from array to Map for allListeners and implemented it strictly.

**Review by pavelfeldman — COMMENTED** [review]
Looks fine, but lint is complaining

**Comment by sand4rt** [comment]
@pavelfeldman can this be merged?

**Review by pavelfeldman — APPROVED** [review]

**Pull request merged**